types/advocate.d.ts

```diff
@@ -0,0 +1,6 @@
+export interface Advocate {
+  image: string,
+  location: string,
+  name: string,
+  region: string
```

Comment:
i feel like this might have to also include `slackId` and `slackUsername` at some point...

Reply:
Good point! 🙂

The `slackId` property was commented out in the `Advocate` interface. I added it to the new interface definition in `types/advocate.ts` as an optional property.

We can then extend is with `slackUsername` when needed.